packages/api/src/brevo/brevo.utils.ts

```diff
@@ -0,0 +1,5 @@
+import * as http from "http";
+
+export function isErrorFromBrevo(error: unknown): error is { response: http.IncomingMessage } {
+    return typeof error === "object" && error !== null && "response" in error && error.response instanceof http.IncomingMessage;
+}
```

Comment:
Usually we say that utils functions outside a service are not nest-like. What is your opinion in such a case, @johnnyomair ?

I think we should create a utils service? Maybe "brevo-api-utils.service.ts"

And then there is a public function in this service which we can use in the brevo-contacts-api.service.ts, brevo-campaigns-api.service.ts and so on

Reply:
Okay, then it's fine for me like it is currently.

We have to improve the error handling for Brevo in the future anyway, I'll make a note of that.